presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -332,6 +337,44 @@ protected void assertCreateTableAsSelect(Session session, String table, @Languag
         assertFalse(getQueryRunner().tableExists(session, table));
     }
 
+    @Test
+    public void testInsertIntoNotNullColumn()
+    {
+        skipTestUnless(supportsNotNullColumns());
+
+        String catalog = getSession().getCatalog().get();
+        assertUpdate("CREATE TABLE test_not_null_with_insert (column_a DATE, column_b DATE NOT NULL)");
```

Comment:
Let's format the test code similar to `TestHiveIntegrationSmokeTest#testShowCreateTable` where we only have the `CREATE TABLE` SQL once in the test.